package.json

```diff
@@ -46,17 +46,17 @@
     "bin",
     "test"
   ],
-  "repository" : { 
-    "type" : "git",
-    "url" : "https://github.com/marcojonker/data-elevator-mongodb.git"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marcojonker/data-elevator-mongodb.git"
   },
-  "homepage" : "http://www.cacadu.eu/opensource",
-  "dependencies" : {
-    "async": "^2.x",
-    "mongodb": "^2.x",
-    "data-elevator": "^2.x"
+  "homepage": "http://www.cacadu.eu/opensource",
+  "dependencies": {
+    "async": "^3.0.1",
+    "mongodb": "^3.2.7",
+    "data-elevator": "^2.2.2"
```

Comment:
This repo depends on `data-elevator`. I think this should be ok, when you would for example publish data-elevator version 2.3.0 or something, right?

Reply:
when data-elevator 2.3.0 is publishen then data-elevator-mongodb should be updated

Reply:
@Kaadude is dit backward compatible met mongodb 2.6?